src/Tribe/Admin/Columns/Tickets.php

```diff
@@ -89,27 +89,31 @@ public function render_column( $column, $post_id ) {
 	 * @return string The column HTML.
 	 */
 	protected function render_tickets_entry( $post_id ) {
-		$output = '—';
+		$output = '&mdash;';
+		$post = get_post( $post_id );
+		$totals = tribe( 'tickets.handler' )->get_post_totals( $post );
 
-		$tickets = Tribe__Tickets__Tickets::get_all_event_tickets( $post_id );
-		if ( empty( $tickets ) ) {
-			return '—';
+		if ( 0 === $totals['tickets'] ) {
+			return $output;
```

Comment:
Why set the `$output` var to `-` and *then* return it only in this case? Do not set `$output` and just return `$mdash;`